lib/presentation/molecules/work_type_selection_molecule.dart

```diff
@@ -20,7 +20,7 @@ class WorkTypeSelectionMolecule extends StatefulWidget {
 
 class _WorkTypeSelectionMoleculeState extends State<WorkTypeSelectionMolecule> {
   late TipType selectedType;
-  bool isTextFinished = false;
+  bool isTextFinished = true;
```

Comment:
That doesn't sound right, the page does not initialize with the text finished rendering in the first run